### DISCUSSION TITLE
Episode 137: Sean Parent on Val (vs Rust)!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#28] [Episode]
Episode Link: https://adspthepodcast.com/2023/07/07/Episode-137.html

Feel free to write comments or ask questions 🙂

**Comment by dabrahams** [comment]
I don't know if you can make corrections but at ~9:08 Sean starts saying Doug Gregor made contributions to Rust, when he meant to say Swift.

**Reply by codereport** [reply]
Can't make audio edit's but I can add a note in the show notes.

**Comment by Brian-M-J** [comment]
At around 3:30 Sean mentions some internal seminars that "will become external seminars". Have they been made public yet? Where can I find them?

**Reply by codereport** [reply]
I took a look on https://stlab.adobe.com/ and nothing stood out. Not sure if @sean-parent will see this but if he does he can comment : )

**Reply by sean-parent** [reply]
A good reminder that I should link mine to my [vanity page](https://sean-parent.stlab.cc/papers-and-presentations/) - but they are here: https://developer.adobe.com/cpp/training/
